packages/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/ProjectConfiguration.kt

```diff
@@ -6,13 +6,17 @@ import com.android.build.gradle.LibraryExtension
 import org.gradle.api.Project
 import org.gradle.internal.extensions.core.extra
 
-private const val defaultCompileSdkVersion = 34
-private const val defaultMinSdkVersion = 23
+private const val defaultCompileSdkVersion = 35
+private const val defaultMinSdkVersion = 24
 private const val defaultTargetSdkVersion = 34
```

Comment:
these version bump are unrelated to this change. if you want to bump the version (which is okay because we have 0.77 on main already), please also bump targetsdkversion to that aligns with 0.77: https://github.com/facebook/react-native/blob/bbbeef6fdb5e39723e04f7df4c9b3e04dcf76d29/packages/helloworld/android/build.gradle#L13